spec/controllers/restaurant_controller_spec.rb

```diff
@@ -0,0 +1,77 @@
+require 'rails_helper'
+
+RSpec.describe RestaurantsController, type: :controller do
+  let(:restaurant_hash) do
+    {
+      name: 'Name',
+      genre: :sushi,
+      rating: 1,
+      accepts_10bis: true,
+      delivery_time: :slow
+    }
+  end
+  let(:keys) { restaurant_hash.keys.to_a }
+  before (:each) do
+    Restaurant.create(restaurant_hash)
+  end
+
+  describe 'validations' do
```

Comment:
`describe` should usually correlate to a specific method. So in this case, you'd want `describe #create` and a separate describe for index `describe #index`